src/commons_codec/transform/aws_dms.py

```diff
@@ -94,6 +94,28 @@ def to_sql(self) -> str:
 
         return sql
 
+    def record_to_update(self) -> str:
+        """
+        Serializes an image to a comma-separated list of column/values pairs
+        that can be used in the `SET` clause of an `UPDATE` statement.
+        Primary key columns are skipped, since they cannot be updated.
+
+        IN
+        {'age': 33, 'attributes': '{"foo": "bar"}', 'id': 42, 'name': 'John'}
+
+        OUT
+        data['age'] = '33', data['attributes'] = '{"foo": "bar"}', data['name'] = 'John'
+        """
+        constraints: t.List[str] = []
+        for column_name, column_value in self.record["data"].items():
+            # Skip primary key columns, they cannot be updated
+            if column_name in self.primary_keys:
+                continue
+
+            constraint = f"{self.DATA_COLUMN}['{column_name}'] = '{column_value}'"
```

Comment:
It should be ok to wrap every column in quotes, since CrateDB already has type information for every column and will auto-cast